_layouts/default.html

```diff
@@ -10,7 +10,7 @@
         {{ content }}
       </div>
 
-      {% include footer.html %}
     </div>
+  {% include footer.html %}
```

Comment:
![image](https://user-images.githubusercontent.com/4805803/64198401-11b94100-ce80-11e9-89cf-83473163bc3c.png)

Your footer changes leave a gap here, can we fix that?